### DISCUSSION TITLE
Failed step "publish" of plugin "@semantic-release/github" Resource not accessible by integration

### DISCUSSION THREAD

**Discussion by mikel21** [semantic-release/semantic-release#2436] [Q&A]
## Current behavior

Publish release to github return error on github action:

RequestError [HttpError]: Resource not accessible by integration
    at /home/runner/.npm/_npx/cdf31b77322f1d44/node_modules/@octokit/request/dist-node/index.js:86:21
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  status: 403

but locally 
curl --user "mikel21" -X POST -H "Accept: application/vnd.github.v3+json" --data "{\"tag_name\":\"v1.0.0\",\"target_commitish\":\"dev\",\"name\":\"v1.0.0\",\"body\":\"Description of the release\",\"draft\":false,\"prerelease\":false,\"generate_release_notes\":false}" https://api.github.com/repos/Joiner2959/nagamaruki/releases
it's work
## Expected behavior

Created release in github

## Environment

- **semantic-release** version: 19.0.2
- CI environment: github
- Plugins used: "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    "@semantic-release/github"
- **semantic-release** configuration: module.exports = {
  branches: [
    "main",
    "dev"
  ],
  plugins: [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    "@semantic-release/github"
  ],
  repositoryUrl: "https://Joiner2959:ghp_blablabla@github.com/Joiner2959/nagamaruki"
};
- CI logs: ![image](https://user-images.githubusercontent.com/9212990/168462706-f056536b-932f-415a-9813-c8b6de874b6e.png)

**Comment by emilyw26** [comment]
@mikel21 were you able to fix this?

**Reply by mikel21** [reply]
No

**Comment by xmnlab** [comment]
I found the same issue here https://github.com/ggpedia/artbox/actions/runs/6015518360/job/16319005504#step:6:44

**Reply by xmnlab** [reply]
this fixed my issue: https://github.com/semantic-release/semantic-release/discussions/2707#discussioncomment-5424275
